chia/full_node/bitcoin_fee_estimator.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import annotations
+
+import logging
+from datetime import datetime
+from typing import Any, Dict
+
+from chia.full_node.fee_estimate_store import FeeStore
+from chia.full_node.fee_estimation import FeeBlockInfo, FeeMempoolInfo
+from chia.full_node.fee_estimator import SmartFeeEstimator
+from chia.full_node.fee_estimator_interface import FeeEstimatorInterface
+from chia.full_node.fee_tracker import FeeTracker
+from chia.types.clvm_cost import CLVMCost
+from chia.types.fee_rate import FeeRate
+from chia.types.mempool_item import MempoolItem
+from chia.types.mojos import Mojos
+from chia.util.ints import uint32, uint64
+
+MIN_MOJO_PER_COST = 5
+
+
+def demo_fee_rate_function(cost: int, time_in_seconds: int) -> uint64:
```

Comment:
unused, there is a similar method in fee_estimator.py that doesn't include cost but is used